android/app/src/main/res/values/styles.xml

```diff
@@ -4,9 +4,7 @@
     <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
         <!-- Customize your theme here. -->
         <item name="android:textColor">#000000</item>
-        <!-- Doing <item name="android:windowIsTranslucent">true</item>
-          as requested by react-native-smart-splash-screen crashes on Oreo
-          because we also request portrait mode. -->
+        <item name="android:windowDisablePreview">true</item>
```

Comment:
What does this new line do?

Reply:
This came from here: https://stackoverflow.com/a/56803562